src/http_api_handler.erl

```diff
@@ -55,19 +52,19 @@ handle_request(<<"GET">>, <<"/api/v1/version">>, json, Req, State) ->
     {Response, Req, State};
 
 handle_request(<<"GET">>, <<"/api/v1/status">>, json, Req, State) ->
-    Response = ergw:system_info(),
-    MappedResponse = lists:map(fun({Key, Value}) ->
-					  {_, K} = lists:keyfind(Key, 1, ?FIELDS_MAPPING),
-					  {K, Value}
-			       end, Response),
-    ResponseMap = maps:from_list(MappedResponse),
-    Result = case maps:find('plmnId', ResponseMap) of
-        {ok, {Mcc, Mnc}} ->
-            maps:update('plmnId', [{mcc, Mcc}, {mnc, Mnc}], ResponseMap);
-        _ ->
-            ResponseMap
-    end,
-    {jsx:encode(Result), Req, State};
+    FieldMap = #{accept_new => 'acceptNewRequests',
+		 node_id    => 'nodeId'},
```

Comment:
Please add new field `nodeId` also into `priv/static/specs/http_api_spec.yaml` for `Status` definitions.